duo_client/client.py

```diff
@@ -583,7 +583,14 @@ def raise_error(msg):
             data = json.loads(data)
             if data['stat'] != 'OK':
                 raise_error('Received error response: %s' % data)
-            return (data['response'], data.get('metadata', {}))
+            response = data['response']
+            metadata = data.get('metadata', {})
+            if not metadata and not isinstance(response, list):
+                metadata = response.get('metadata', {})
+                if metadata:
+                    response_key = [key for key in response.keys()if key != 'metadata'].pop()
```

Comment:
A comment here might be nice explaining that if `metadata` is inside `response`, we're assuming (based on precedent of endpoints doing this) that:

 # There will only be one sibling property to `metadata` inside `response`
 # That sibling property will have a list value containing the items to be iterated

Reply:
Also if we _really_ wanted to be robust in this case, we could set `response_key` to the first key that has a list value. In theory it should always be a list so that check might be overkill. Your call.